js/views/buttonsView.js

```diff
@@ -185,13 +186,19 @@ export default class ButtonsView extends Backbone.View {
 
     const $ariaLabel = this.$('.js-btn-marking-label');
     const hasSpanAriaLabel = Boolean($ariaLabel.length);
+    const correctnessAriaLabel = isCorrect
+      ? ariaLabels.answeredCorrectly
+      : isPartlyCorrect
+        ? ariaLabels.answeredPartlyCorrect
```

Comment:
```suggestion
        ? (ariaLabels.answeredPartlyCorrect ?? ariaLabels.answeredIncorrectly)
```
Just for backward compatibility?

If `ariaLabels.answeredPartlyCorrect` is `null` or `undefined` then use `ariaLabels.answeredIncorrectly`.

Reply:
Thanks @oliverfoster. Amended in [ec043c4](https://github.com/adaptlearning/adapt-contrib-core/pull/554/commits/ec043c48a965d48b1de342fd1ebc1b557d158e08).